tests/drivers/adc/adc_api/src/test_adc.c

```diff
@@ -10,6 +10,21 @@
 #include <zephyr.h>
 #include <ztest.h>
 
+/* ADC measurement of floating input is not guaranteed to provide non zero value
+ * Nevertheless, it works. But for some boards, a delay may be required
+ * between 2 consecutive measurements of a floating input.
+ * Interval in microseconds.
```

Comment:
I think I don't entirely understand the motivation behind these changes. Cannot this input be just wired to something that would give the non-zero result, instead of being left floating?

Reply:
The board is on a test bench.
In order to pass the test "test/drivers/dac/dac_loopback",  there is already a wire between dac and adc.
If I add a wire, lets say to 3.3V, for the test "tests/drivers/adc/adc_api", then the test "test/drivers/dac/dac_loopback" will be failed.

Reply:
Okay, I see.